src/libraries/Microsoft.Extensions.Primitives/src/Microsoft.Extensions.Primitives.csproj

```diff
@@ -11,6 +11,8 @@ Commonly Used Types:
 Microsoft.Extensions.Primitives.IChangeToken
 Microsoft.Extensions.Primitives.StringValues
 Microsoft.Extensions.Primitives.StringSegment</PackageDescription>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
To anyone else reviewing this: The Microsoft.Extensions assemblies all have IsPackable set to true outside of the csproj.

Reply:
Yep - https://github.com/dotnet/runtime/blob/061c8b7177fbe56584994325a24bb262628ebd5a/src/libraries/Directory.Build.props#L79